.travis.yml

```diff
@@ -39,9 +39,11 @@ after_success:
 
 deploy:
   provider: surge
-  project: ./dist
-  domain: https://development-test-LoklakSearch.surge.sh
-  on: development
+  skip_cleanup: true
```

Comment:
Why we use `skip_cleanup: true` ?

Reply:
Please refer: https://github.com/AhsanAyaz/ng-traffic-ci/blob/master/.travis.yml and https://docs.travis-ci.com/user/deployment